roadmap/implementors-guide/guide.md

```diff
@@ -571,48 +578,48 @@ It aims to achieve these tasks with these goals in mind:
 
 The Scheduler manages resource allocation using the concept of "Execution Cores". There will be one execution core for each parachain, and a fixed number of cores used for multiplexing parathreads. Validators will be partitioned into groups, with the same number of groups as execution cores. Validator groups will be assigned to different execution cores over time.
 
-An execution core can exist in either one of two states at the beginning or end of a block: free or occupied. A free execution core can have a parachain or parathread assigned to it for the potential to have a backed candidate included. After inclusion, the core enters the occupied state as the backed candidate is pending availability. There is an important distinction: a core is not considered occupied until it is in charge of a block pending availability, although the implementation may treat scheduled cores the same as occupied ones for brevity. A core exits the occupied state when the candidate is no longer pending availability - either on timeout or on availability. A core starting in the occupied state can move to the free state and back to occupied all within a single block, as availability bitfields are processed before backed candidates. At the end of the block, there is a possible timeout on availability which can move the core back to the free state if occupied. 
+An execution core can exist in either one of two states at the beginning or end of a block: free or occupied. A free execution core can have a parachain or parathread assigned to it for the potential to have a backed candidate included. After inclusion, the core enters the occupied state as the backed candidate is pending availability. There is an important distinction: a core is not considered occupied until it is in charge of a block pending availability, although the implementation may treat scheduled cores the same as occupied ones for brevity. A core exits the occupied state when the candidate is no longer pending availability - either on timeout or on availability. A core starting in the occupied state can move to the free state and back to occupied all within a single block, as availability bitfields are processed before backed candidates. At the end of the block, there is a possible timeout on availability which can move the core back to the free state if occupied.
```

Comment:
We say inclusion for backing everywhere?  It might prove confusion since backing need not result in execution.  I avoided this issue by saying "backed/backing" everywhere, but that's easier when you're writing primarily about the security.  We could say backing announcement or announced for short?

Reply:
I called these availability cores because what they actually do is mostly availability.  It's true they cause the execution of the parablock when availability is reached, and saying execution helps clarifies that, and maybe the timing of execution is the more distinctive feature of the protocol.  I think names like execution-availability core or availability-through-execution core, which shorten how you like work too.

Reply:
(all of this is just showing up in the diff due to removed line endings, so it's beyond scope of this PR but still worth addressing)

I like "availability core" better than execution core. That seems beyond the scope of the module. I would accept a PR changing all instances of execution core to availability core. 

Yeah, I think we need to clean up terminology a bit. The problem I faced was that it seems that something can be backed (i.e. have a complete backing) without being included on-chain. So I always jump to calling something "included" wrongly. We should have terminology for:
  - backed but off-chain
  - backed but on-chain (specifically, on _this_ chain)
  - available (included?) but not past acceptance period
  - available & accepted

Reply:
Yes, I agree backed could mean off-chain, oops.  I actually wrote "backing attestation" or now maybe "backing statement" for what could exist off-chain, and wrote backed for on-chain, but yes that's too confusing.  So maybe:
 - backing statement(s) might mesh well with the existing document, although these statements get gossiped partially only within parachain validator groups
 - backing announced or backing included, with then announced or included for short,
 - availability votes/statements,
 - available/executed,
 - acceptance statements/attestations, 
 - accepted.

I expected you'd want some explicit intrinsic? declaring availability probably.  I now recall some comment about doing that computation implicitly, which makes "included" make more sense.  I'd thought that'd make polkadot less lite client friendly, but maybe it's all computations from the state root anyways, so if you've the proof then you're fine.

Reply:
How about this? 
  - potentially-backed (off-chain)
  - backed = pending-availability (on-chain, only makes sense when talking about a specific fork)
  - available = included (on-chain, considered available, not yet accepted)
  - accepted (on-chain, included for the entire acceptance period without losing dispute)